stemcell_builder/stages/untar_base_os_image/apply.sh

```diff
@@ -6,7 +6,7 @@ base_dir=$(readlink -nf $(dirname $0)/../..)
 source $base_dir/lib/prelude_apply.bash
 
 # the tar_base_os_image stage creates a tar ball of the chroot with no enclosing directory
-tar zxf $os_image_tgz -C $chroot
+sudo tar zxf $os_image_tgz -C $chroot --same-owner
```

Comment:
Per the [tar man page](https://linux.die.net/man/1/tar), `--same-owner` is the default behavior for the superuser, so don't need to specify that flag.